packages/dev/core/src/Misc/tools.ts

```diff
@@ -842,25 +766,25 @@ export class Tools {
 
     /**
      * Encodes the canvas data to base 64 or automatically download the result if filename is defined
+     * @param canvas canvas to get the data from.
      * @param successCallback defines the callback triggered once the data are available
      * @param mimeType defines the mime type of the result
      * @param fileName defines he filename to download. If present, the result will automatically be downloaded
-     * @param canvas canvas to get the data from. If not provided, use the default screenshot canvas
      * @param quality defines the quality of the result
      */
     static EncodeScreenshotCanvasData(
+        canvas: HTMLCanvasElement,

```

Comment:
This one is a breaking change? Though I don't know if a lot of people are calling `EncodeScreenshotCanvasData` directly...

Reply:
It is. Was there a need to change the signature?

Reply:
The other function was taking a temp canvas by default which could lead to a lot of weird side effects :-) I ll change it back if the community asks for it but pretty sure never used externally